assertions.js

```diff
@@ -0,0 +1,116 @@
+/*!
+ * Copyright (c) 2024 Digital Bazaar, Inc.
+ */
+import chai from 'chai';
+
+const should = chai.should();
+// RegExp with bs58 characters in it
+const bs58 =
+  /^[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]+$/;
+
+/**
+ *  Regex to test conformity to base-64-url-no-pad character set.
+ *
+ * @see {@link https://www.w3.org/TR/vc-data-integrity/#multibase-0} for
+ * specification.
+*/
+const BASE_64URL_NOPAD_REGEX = /^[A-Za-z0-9\-_]+$/;
+
+// assert something is entirely bs58 encoded
+export const shouldBeBs58 = s => bs58.test(s);
+
+export const shouldBeBase64NoPadUrl = s => BASE_64URL_NOPAD_REGEX.test(s);
+
+export const verificationFail = async ({
+  credential, verifier, options = {}
+} = {}) => {
+  const body = {
+    verifiableCredential: credential,
+    options: {
+      checks: ['proof'],
+      ...options
+    }
+  };
+  const {result, error} = await verifier.post({json: body});
+  should.not.exist(result, 'Expected no result from verifier.');
+  should.exist(error, 'Expected verifier to error.');
+  should.exist(error.status, 'Expected verifier to return an HTTP Status code');
+  error.status.should.equal(
+    400,
+    'Expected HTTP Status code 400 invalid input!'
+  );
+};
+
+export function expectedMultibasePrefix(cryptosuite) {
+  const b64urlNoPadSuites = ['ecdsa-sd-2023', 'bbs-2023'];
+
+  if(b64urlNoPadSuites.includes(cryptosuite)) {
+    return {
+      prefix: 'u',
+      name: 'base64url-no-pad'
```

Comment:
Per the constant conversation above...just "match" these names for the constants: `base64url_no_pad_regex` (perhaps).

Reply:
we do need to stick with a convention for sure, this is used here: https://github.com/w3c-ccg/data-integrity-test-suite-assertion/blob/e6ecde60b53e37a1fc46333b5840896ffb17cd82/suites/create.js#L173-L174

So changing the names of the bs58 and base64UrlNoPad in this PR would be major. Perhaps just an issue for now?

Reply:
Yeah...just make an issue...and lets monitor this going forward. Thanks.

Reply:
issue made: https://github.com/w3c-ccg/data-integrity-test-suite-assertion/issues/55